src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -52,34 +57,33 @@ export const useSwapper = () => {
     Trade<KnownChainIds>,
   ]
   const adapterManager = useChainAdapters()
-  const [swapperManager] = useState<SwapperManager>(() => {
-    const manager = new SwapperManager()
-    return manager
-  })
+  const [swapperManager] = useState<SwapperManager>(() => new SwapperManager())
 
   useEffect(() => {
     if (!adapterManager || !swapperManager) return
 
     const web3 = getWeb3Instance()
 
-    ;(async () => {
-      // const midgardUrl = getConfig().REACT_APP_MIDGARD_URL
-      // const thorSwapper = new ThorchainSwapper({
-      //   midgardUrl,
-      //   adapterManager,
-      //   web3,
-      // })
-      // await thorSwapper.initialize()
-      // swapperManager.addSwapper(SwapperType.Thorchain, thorSwapper)
-
-      const zrxSwapper = new ZrxSwapper({
-        web3,
-        adapter: adapterManager.get('eip155:1') as unknown as ethereum.ChainAdapter,
-      })
-
-      await zrxSwapper.initialize()
+    // ;(async () => {
+    //   const midgardUrl = getConfig().REACT_APP_MIDGARD_URL
+    //   const thorSwapper = new ThorchainSwapper({
+    //     midgardUrl,
+    //     adapterManager,
+    //     web3,
+    //   })
+    //   await thorSwapper.initialize()
+    //   swapperManager.addSwapper(SwapperType.Thorchain, thorSwapper)
+    // })()
```

Comment:
Can we add a comment on l.66 explaining why this whole block is commented out?

Reply:
I'd only be guessing, but I assume @elmutt did the `thorSwapper` implementation and then commented it out until we needed it.

@elmutt can you confirm?

Reply:
correct

Reply:
https://github.com/shapeshift/web/pull/2107/commits/1396be4d1b69881217749260a0754b006b3fec38